doc/overview.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Overview of dbxincluder
+=======================
+
+dbxincluder is an implementation of the XInclude 1.1 specification (https://www.w3.org/TR/xinclude-11)
+with support for DocBook transclusion (http://docbook.org/docs/transclusion).
+
+
+Limitations
+===========
+
+It does not support the XInclude 1.1 "xpointer" attribute, as the XPointer functionality is currently not exposed by lxml.
```

Comment:
Maybe the word "exposed" carries a lot of baggage and we interprete it in different ways. The lxml library can access (expose?) the `xpointer` attribute of course:

```python
xml = etree.fromstring("""<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="foo.xml" xpointer="foofoo"/>""")
print("xpointer:", xml.attrib['xpointer'])
```
would print `'foofoo`. So generally it works. What do you mean by "does not expose that functionality"?